ssas/service/admin/api_test.go

```diff
@@ -82,6 +82,31 @@ func (s *APITestSuite) TestCreateGroup() {
 	assert.Nil(s.T(), err)
 }
 
+func (s *APITestSuite) TestListGroups() {
+	groupBytes := []byte(SampleGroup)
+	gd := ssas.GroupData{}
+	err := json.Unmarshal(groupBytes, &gd)
+	assert.Nil(s.T(), err)
+	g1, err := ssas.CreateGroup(gd)
+	assert.Nil(s.T(), err)
+
+	gd.ID = "some-fake-id"
+	gd.Name = "some-fake-name"
+	g2, err := ssas.CreateGroup(gd)
+	assert.Nil(s.T(), err)
+
+	req := httptest.NewRequest("GET", "/group", nil)
+	handler := http.HandlerFunc(listGroups)
+	rr := httptest.NewRecorder()
+	handler.ServeHTTP(rr, req)
+	assert.Equal(s.T(), http.StatusOK, rr.Result().StatusCode)
+	assert.Equal(s.T(), "application/json", rr.Result().Header.Get("Content-Type"))
```

Comment:
Should this also be checking the body of the response?

Reply:
sure, why not! added that check to the test